### ISSUE TITLE
[BUG] Error: illegal base64 data at input byte 0

### ISSUE THREAD

**Issue by wwhiffle** [juzeon/SydneyQt#172]
### Checks

- [x] I confirm that I have searched existing issues and couldn't find a similar bug.
- [x] I have read the documentation carefully and it couldn't resolved my problem.

### Describe the bug

Two weeks ago, I was still able to use SydneyQt normally. After a lapse of two weeks, when I opened SydneyQt again, it popped up an error window. I am not clear what triggered this error, as there were no changes to SydneyQt during these two weeks. I tried downloading the latest version of SydneyQt and placed it in a separate empty folder, but the same error window still pops up. Perhaps the problem lies with my Windows system. I would be very grateful if you could provide some advice.

```
Error: illegal base64 data at input byte 0
Details: file(D:/a/SydneyQt/SydneyQt/app.go), line(90).
Instruction: This is probably an unknown bug. Please take a screenshot and report this issue.
```

### To Reproduce

Perhaps this bug only occurs in my Windows, because of the conditions, I don't have extra devices to reproduce it.

### Expected behavior

I hope that when opening SydneyQt, no error pop-up will appear, and I can chat as usual.

### Screenshots

<img width="890" alt="PixPin_2023-12-26_10-39-05" src="https://github.com/juzeon/SydneyQt/assets/100055123/6e5f0ad8-9284-4df4-8a40-b6055cec2c60">

### Desktop (please complete the following information):
 - OS and Version: Windows 10
 - SydneyQt Version: v2.2.4

### Additional context

none.

**Comment by juzeon** [comment]
The panic is produced by:
```diff
var tk *tiktoken.Tiktoken
var initTkFunc = sync.OnceFunc(func() {
	slog.Info("Init tiktoken")
	t, err := tiktoken.EncodingForModel("gpt-4")
	if err != nil {
+ 		util.GracefulPanic(err)
	}
	tk = t
})
```

Seems when initializing the tiktoken-go library, which has not been touched however. Need more investigation. Would you please set up a virtual machine and see if this still occurs?

**Comment by wwhiffle** [comment]
Based on your suggestion, I have created a VMware virtual machine with a Windows 10 system. In the virtual machine, I successfully opened SydneyQt and started chatting as usual.

That is to say, I can now successfully use SydneyQt in the Windows 10 system within the virtual machine, but I cannot use it in the original Windows 10 system.

That's the current situation.

**Comment by juzeon** [comment]
Weird I have to say. So what about running via `wails dev`?

**Comment by wwhiffle** [comment]
I'm not sure if I've understood you correctly, as I'm not very familiar with this kind of technology.

I tried to run `wails dev` in the root directory of the project following the steps in the README. After running this command, SydneyQt automatically runs and still pops up an error window.

I've copied all the output text in the terminal to a txt file, and here is the link to the txt file. I hope this will help to solve the problem.

[VSCode Terminal Output Text.txt](https://github.com/juzeon/SydneyQt/files/13772176/VSCode.Terminal.Output.Text.txt)
